src/test/obj_tx_add_range/TEST0

```diff
@@ -40,7 +40,10 @@ export UNITTEST_NUM=0
 # standard unit test setup
 . ../unittest/unittest.sh
 
-require_test_type medium
+require_test_type	medium
+require_fs_type		pmem non-pmem
+
+configure_valgrind pmemcheck force-enable
```

Comment:
there usually should be at least one test with no extra tools, and this one was it ;)
The TEST1 in this directory already has pmemcheck force enabled.